docs/openj9_newuser.md

```diff
@@ -0,0 +1,100 @@
+- **java.lang.management API:** OpenJ9 provides MXBean additions and extensions to this standard API, which enables you to use tools such as JConsole to monitor and manage your Java applications. For more information, see [MBeans and MXBeans](https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/mxbeans.html).
+
+If you are familiar with using HotSpot as part of an Oracle JDK or OpenJDK, you might make use of certain `com.sun.management` interfaces. Although OpenJ9 implements some of these interfaces, a few are specific to the HotSpot VM. The following table indicates alternative classes or mechanisms that you can use for equivalent function in OpenJ9:
+
+| Hotspot-specific classes   | Alternatives for OpenJ9  |
+|----------------------------|--------------------------|
+|`HotSpotDiagnosticMXBean`   | `OpenJ9DiagnosticsMXBean`|
+|`MissionControl`            | Use Health Center        |
+|`MissionControlMXBean`      | Use Health Center        |
+|`ThreadMXBean`              | `JvmCpuMonitorMXBean` (for thread time)    |
+|`VMOption`                  | OpenJ9 Java dump (option `-Xdump:java`) |
+|`DiagnosticCommandMBean`    | None                     |
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** OpenJ9 implements the following `com.sun.management` interfaces: `GarbageCollectorMXBean`, `GarbageCollectionNotificationInfo`, `GcInfo`, `OperatingSystemMXBean`, `UnixOperatingSystemMXBean`.
+
+## Other differences
+
+This topic describes the differences between the HotSpot VM and the Eclipse OpenJ9 VM. Therefore, if you are currently using an OpenJDK with the default Hotspot VM and you want to switch to using an OpenJDK with the OpenJ9 VM, these are the only differences you might be concerned about. If however, you are using an Oracle JDK, you might want to learn about differences between other components that make up an Oracle JDK or an OpenJDK from the AdoptOpenJDK community.
+
+The following proprietary features are not available in an OpenJDK v8 binary:
+
+**JavaFX:**
+
+This feature is deprecated by Oracle in Java SE 9 and is not available in Oracle JDK 11. The source code was contributed to OpenJDK under the OpenJFX project.
+
+**Java plug-in and webstart:**
+
+These features were deprecated by Oracle in Java SE 9 and are not available in Oracke JDK 11. Oracle encourages developers to migrate to other solutions.
+
+**T2K font library:**
+
+T2K is a proprietary native font library, which is licensed for use by Java 2D from a third party. OpenJDK uses the open FreeType font library as an alternative. The rendering of these fonts is almost identical and situations where there are differences are rare.
+
+Discussions are underway at the AdoptOpenJDK project to include OpenJFX and investigate the IcedTea-web project as a replacement for the Web Start capabilities.
```

Comment:
This paragraph looks like it is part of the `T2K font library` topic, but it is related to JavaFX.